vllm/worker/habana_model_runner.py

```diff
@@ -1938,14 +1938,16 @@ def execute_model(
         return [output]
 
     def shutdown_inc(self):
-        print('inc shutdown')
-        if (model_config := getattr(self, "model_config", None)) and \
-                         getattr(model_config, "quantization", None) == 'inc':
-            print('inc shutdown start')
+        is_inc_used = (model_config :=
```

Comment:
AFAIK getattr can handle None. Could we chain the calls like this:
```python
is_inc_used = getattr(getattr(self, "model_config", None), "quantization", None) is None
...
is_model_initialized = getattr(getattr(self, "model", None), "model", None) is None

```
?

Reply:
Or even better:
https://stackoverflow.com/a/3283072

Reply:
fixed, thanks!